### PR TITLE
Regain everything back in FUN

### PR DESCRIPTION
**Pull request by janaSunrise**

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Remove the HUGE help string, we will setup a custom help command later, with a paginator.

File: cogs/coding.py [hunk 1]
```diff
@@ -85,31 +85,11 @@ def get_h2_content(self, tag: NavigableString) -> str:
...
-        Useful to hide your syntax fails or when you forgot to print the result.
-        """,
-        brief="Execute code in a given programming language",
-    )
+    @commands.command()
     async def run(self, ctx: Context, language: str, *, code: str = "") -> t.Union[None, str]:
         """Execute code in a given programming language."""
 
-        options = {"--stats": False, "--wrapped": False}  # the flags to be used when the compiler is needed
+        options = {"--stats": False, "--wrapped": False}  # the flags to be used when the compler is needed
```
Comment (ItsDrike) [inline cogs/coding.py:92]: How is this ok? Why did you change `compiler` to `compler` that doesn't seem gramatically correct.

File: cogs/coding.py [hunk 2]
```diff
@@ -217,8 +196,7 @@ async def run(self, ctx: Context, language: str, *, code: str = "") -> t.Union[N
                 if matches:
                     message = message + f" Maybe you meant {matches}?"  # provide a suggestion.
```
Comment (ItsDrike) [inline cogs/coding.py:197]:
```suggestion
                    message +=  f" Maybe you meant {matches}?"  # provide a suggestion.
```

**Review by ItsDrike — CHANGES_REQUESTED** [review]

**Review by ItsDrike — COMMENTED** [review]
This can be merged after #62 and after all requested changes are resolved
